CMakeLists.txt

```diff
@@ -51,8 +51,12 @@ find_package(folly REQUIRED)
 find_package(wangle REQUIRED)
 find_package(mvfst REQUIRED)
 find_package(proxygen REQUIRED)
-# We can NOT use Glog latest version, is NOT compatible with moxygen
-find_package(glog REQUIRED)
+# We can NOT use Glog latest version, is NOT compatible with moxygen - XXX is this comment still relevant
+# find_package(glog REQUIRED) - XXX this fails OSS build
```

Comment:
@lnicco asks:

I guess I don't fully understand why for the author find_package would fail but pkg_check_modules passes.